### PR TITLE
Move __version__ to style submodule

### PR DESCRIPTION
**Pull request by zurk**
As was discussed this `__version__` related to style-analyzer part and for the lookout-core part will be another package and module( see issue https://github.com/src-d/style-analyzer/issues/266)

### CODE REVIEW

**Comment by zurk** [comment]
but it is in the wrong place even now. this `__init__` can be overwritten by `lookout.sdk` (https://github.com/src-d/lookout-sdk) package.

**Comment by m09** [comment]
Indeed. Then I'd rather have a `__version__.py` file in the `lookout` folder in the meantime. It's just weird to have a file affecting two packages but placed in only one of them.

File: lookout/core/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# This is version for further lookout-core python package.
+# It should be used after core+format splitting and this comment should be deleted.
+# If you want to bump version now go to lookout.style.__init__.__version__
+__version__ = (0, 0, 1)
```
Comment (vmarkovtsev) [inline lookout/core/__init__.py]: Should not be braces

File: lookout/style/__init__.py [hunk 2]
```diff
@@ -1,2 +1,3 @@
 """Supported machine learning based analyzers for lookout."""
+__version__ = 0, 0, 3
```
Comment (vmarkovtsev) [inline lookout/style/__init__.py]: Can you also please remove `__meta__` because damn it is only needed in the tests as far as i remember

Comment (vmarkovtsev) [reply lookout/style/__init__.py]: Shit we need to document that

**Pull request merged**